### PR TITLE
#170663818 Users should be able to create multicity trip requests 

### PR DESCRIPTION
**Pull request by NiyongaboEric**
#### What does this PR do?
PR helps users to create multicity trip requests.

#### Description of Task to be completed?
**Given** users are visiting private pages 
**When** they did not authenticate
**Then** we should be able to redirect them to login page

#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-multicity-trip-170663818`
- `npm install` packages
- `npm run dev`
-  Login to the system
-  Visit `URL/trips/multicity`

#### What are the relevant pivotal tracker stories? 
[#170663818](https://www.pivotaltracker.com/story/show/170663818)

#### Screenshots (if appropriate)
Screen Shot 2020-01-22 at 17 57 24

#### Questions:
N/A

### CODE REVIEW

**Review by victkarangwa — COMMENTED** [review]
This is really nice work @NiyoEric.
But you can make it better if you add navigation menu for the users to access the page
Screen Shot 2020-01-23 at 16 59 54
 directly

**Comment by victkarangwa** [comment]
And also work on the tests, as of now they are failing

File: src/__tests__/actions/Trips/multicity.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.reject(errorResp);
+    });
+
+    const store = mockStore({});
+    await store.dispatch(sendMulticityTrip(multicityTripAction.multicityObjectData))
+      .catch((error) => {
+      
+        console.log(error, error.status, '<---------->', errorResp.status);
```
Comment (hezronkimutai) [inline src/__tests__/actions/Trips/multicity.js]: Remove this console here.

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
You have done a good job @NiyoEric , I left a comment, have a look at it, and . also work on the responsiveness of the app and we will be good to go.

NB: to ensure that the tests in circle.ci passes, run it several times till it passes.

File: src/containers/Trips/MulticityTrip.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  }
+
+  render() {
+    const { multicityTripData, viewProfile } = this.props;
+    return (
+      <Dashboard>
+        {viewProfile.spinner && <Spinner/>}
+        <div className="oneway-container">
+          <TripHeader />
+          { multicityTripData.message && <Redirect to="/requests" /> }
```
Comment (hezronkimutai) [inline src/containers/Trips/MulticityTrip.js:65]: I prefer you redirect the user to specific request i.e
`<Redirect to="/requests/:tripRequestId" />  `

**Review by hezronkimutai — APPROVED** [review]

**Review by shemaeric — APPROVED** [review]
Good work @NiyongaboEric

**Pull request merged**